src/lib/Components/Inbox/ActiveBids/index.tsx

```diff
@@ -20,6 +38,27 @@ class ActiveBids extends React.Component<RelayProps, null> {
     return bids
   }
 
+  refreshActiveBids(callback) {
+    if (this.state.fetchingData) {
+      return
+    }
+
+    this.setState({ fetchingData: true })
```

Comment:
I don’t believe any of this type of state is still necessary, you can probably just check `this.props.relay.isLoading()`